src/terminal/osc.zig

```diff
@@ -158,6 +158,9 @@ pub const Command = union(enum) {
     /// End a hyperlink (OSC 8)
     hyperlink_end: void,
 
+    /// Sleep (OSC 9;1) in ms
+    sleep: u16,
```

Comment:
I considered using the [Duration type](https://github.com/ghostty-org/ghostty/blob/94599102e9fb8247af08cbbbcb7ee25e3d31e1bd/src/config/Config.zig#L5546) but this would introduce a dependency on Config. Would that be an issue?

Reply:
I don't think that the Duration type is necessary here, unless there's a need to display it with units or parse it from a string units. You might consider scaling it to nanoseconds and using a `u64` since that's what Zig tends to use for that sort of thing.

Reply:
Since the duration is in the ConEmu spec as `ms` I'm fine keeping this as `u16` and documenting it (as @dmehala already has) as a millisecond unit.